### DISCUSSION TITLE
Intro to GraphRAG: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#55] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/graph
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=f6pUqDeMiG0

Slides: 
[PDF](https://github.com/ms-johnalex/intro-to-graphrag/blob/main/RAGHackSession-Intro-to-GraphRAG.pdf)
[PPTX](https://github.com/microsoft/RAG_Hack/blob/main/slides/RAGHackSession-Intro-to-GraphRAG.pptx)

John's Code Samples: https://github.com/ms-johnalex/intro-to-graphrag

Please ask any follow-up questions here, and the speakers will see them.

**Comment by Yash-Kavaiya** [comment]
How can we apply Graph RAG using Github Models ?

**Reply by moapurva** [reply]
Can you be more specific about what you mean by GitHub Models?

**Reply by pamelafox** [reply]
I'm guessing Yash is referring to https://github.com/marketplace/models
They are freely usable models, but they have very low rate limits, so I doubt they can be used for GraphRAG. I only use them when testing single calls to the APIs, I never use them for multiple call situations like evaluation or processing.

**Reply by moapurva** [reply]
Thanks Pamela. @Yash-Kavaiya, we recommend using only GPT-4 or newer models, and while GPT-4o-mini is available, the rate limits are too low. https://docs.github.com/en/github-models/prototyping-with-ai-models#rate-limits.

**Comment by shyshin** [comment]
As a follow up to your answer on using single domain input for graph RAG, how do we go about graph updation? when GraphRAG app eventually goes into production. Do we index everything again?

**Reply by moapurva** [reply]
Incremental update functionality is on the roadmap. Until then, start with smaller datasets to validate proper graph creation before adding more data.
